api/Document.json

```diff
@@ -4629,24 +4631,11 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": [
-              {
-                "version_added": "41"
-              },
-              {
-                "version_added": "35",
-                "version_removed": "41",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.font-loading-api.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              }
-            ],
+            "firefox": {
+              "version_added": "41"
+            },
```

Comment:
Hey @foolip and @vinyldarkscratch — I'm looking back through some recently merged PRs, just to get a sense of how things are going. Since the flag removal guideline is new, I'm keeping on eye on that sort of thing, to make sure it's working out as we'd hoped.

It looks like this was removed to be consistent with Android. I'm guessing this is a reverse mirroring, where Firefox for Android data overwrote the desktop data. Is that the whole story?

More broadly, it'd be helpful to me if PRs would at least note that some data was removed and what the motivation was for it, at least in the short term. That might spare me some having to inquire about already-merged PRs. 😄

Reply:
This came from me pushing back against adding more old flag data in https://github.com/mdn/browser-compat-data/pull/6818#discussion_r499510959 and suggesting removing the existing data instead.

@vinyldarkscratch when we remove flag data going forward, can you update the PR description? I've recently been using the PR description as the commit message, so it would then show up there too.

Reply:
Thank you for the explanation on this. And thanks for filling me on how this happened. Seeing the comment in context, it does strike me as… uninteresting to replicate old flag data. This is fine. I do want to continue to follow our use of the new guideline though—thank you for any effort to keep me in the loop on future removals. 👍

(My inbox is in a dire state apparently. Sorry for only just getting to this now.)